services/visual_recognition/v3.js

```diff
@@ -86,12 +75,7 @@ module.exports = function (RED) {
   // it should be called here or after alchemy returns and passed
   // control back to cbdone.
 
-  function performAction(params, feature, cbdone) {
-    var visualRecognition = watson.visual_recognition({
-      api_key: apikey,
-      version: 'v3',
-      version_date: '2016-05-19'
-    });
+  function performAction(visualRecognition, params, feature, cbdone) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'performAction' has a complexity of 8.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2712988163/issues/source?bid=3330694&fileBranchId=3344618#l78)